components/d2l-applied-filters/d2l-applied-filters.js

```diff
@@ -0,0 +1,227 @@
+import { LitElement, css, html } from 'lit-element';
+import { LocalizeStaticMixin } from '@brightspace-ui/core/mixins/localize-static-mixin.js';
+
+import '@brightspace-ui-labs/multi-select/multi-select-list';
+import '@brightspace-ui-labs/multi-select/multi-select-list-item';
+import { bodyCompactStyles } from '@brightspace-ui/core/components/typography/styles.js';
+import { getComposedChildren } from '@brightspace-ui/core/helpers/dom';
+
+const DROPDOWN_NAME = 'D2L-FILTER-DROPDOWN';
+const DROPDOWN_CATEGORY_NAME = 'D2L-FILTER-DROPDOWN-CATEGORY';
+const DROPDOWN_OPTION_NAME = 'D2L-FILTER-DROPDOWN-OPTION';
+
+class D2lAppliedFilters extends LocalizeStaticMixin(LitElement) {
+	static get properties() {
+		return {
+			for: { type: String },
+			_entries: { type: Array },
+			_selectedEntries: { type: Array },
+			_target: { type: Object },
+		};
+	}
+
+	static get styles() {
+		return [bodyCompactStyles, css`
+			:host {
+				display: block;
```

Comment:
Minor: For web components with a default `display` that's different than `inline`, it's [good practice](https://developers.google.com/web/fundamentals/web-components/best-practices#add-a-:host-display-style-that-respects-the-hidden-attribute.) to add handling for the `hidden` attribute:
```
:host([hidden]) {
  display: none
}
```